website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -90,6 +92,85 @@ metrics:
 
 </File>
 
+<VersionBlock firstVersion="1.2">
+
+### Available properties
+
+| Field       | Description                                                 | Example                         | Required? |
+|-------------|-------------------------------------------------------------|---------------------------------|-----------|
+| name        | A unique identifier for the metric                          | new_customers                   | yes       |
+| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
+| label       | A short for name / label for the metric                     | New Customers                   | no        |
+| description | Long form, human-readable description for the metric        | The number of customers who.... | no        |
+| type        | The type of calculation to perform when evaluating a metric | count_distinct                  | yes       |
+| sql         | The expression to aggregate/calculate over                  | user_id                         | yes       |
+| timestamp   | The time-based component of the metric                      | signup_date                     | yes       |
+| time_grains | One or more "grains" at which the metric can be evaluated   | [day, week, month]              | yes       |
+| dimensions  | A list of dimensions to group or filter the metric by       | [plan, country]                 | no        |
+| filters     | A list of filters to apply before calculating the metric    | See below                       | no        |
+| meta        | Arbitrary key/value store                                   | {team: Finance}                 | no        |
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.2">
+
+### Available types
```

Comment:
Any reason why we can't show this table (or at least most of it) to users of v1.0 + v1.1 as well? I'd propose removing the `VersionBlock` wrapping around the whole table, and then putting something like this for the last line:
```
| expression | <VersionBlock firstVersion="1.2">This metric type is defined as any **non-aggregating** calculation of 1 or more metrics</VersionBlock><VersionBlock lastVersion="1.1">Not yet available — added in v1.2</VersionBlock> |
```